src/main.cc

```diff
@@ -401,7 +401,9 @@ std::vector<Option> options_
        "The port for the server to listen on for HTTP requests."},
       {OPTION_REUSE_HTTP_PORT, "reuse-http-port", Option::ArgBool,
        "Allow multiple servers to listen on the same HTTP port when every "
-       "server has this option set."},
+       "server has this option set. The same set of models/same model "
```

Comment:
"If you plan to use this option as a way to load-balance between different triton servers, the same model repository or set of models must be used for every server."


> Note that this feature only supports stateless models.

I think it might be better to remove this sentence since the customers may figure out a way to control which server gets the requests and address this limitation.

Reply:
I see. Updated the document, thanks for the comment!